content/en/docs/reference/networking/ports-and-protocols.md

```diff
@@ -27,6 +27,7 @@ etcd cluster externally or on custom ports.
 | Protocol | Direction | Port Range  | Purpose               | Used By                 |
 |----------|-----------|-------------|-----------------------|-------------------------|
 | TCP      | Inbound   | 10250       | Kubelet API           | Self, Control plane     |
+| TCP      | Inbound   | 10256       | kube-proxy            | Self                    |
```

Comment:
Does anything other than kube-proxy connect to this port? (A load balancer, perhaps?)

Reply:
Indeed, updating now